packages/components/select/src/useSelect.ts

```diff
@@ -595,7 +595,7 @@ export const useSelect = (props: ISelectProps, emit) => {
   }
 
   const getValueIndex = (arr: any[] = [], option) => {
-    if (!isObject(option?.value)) return arr.indexOf(option.value)
+    if (!isObject(option?.value)) return arr.indexOf(option?.value)
```

Comment:
Maybe we can add a check directly, and return -1 directly if option is `undefiend`. 🤔